src/match_identifiers.cpp

```diff
@@ -41,7 +54,7 @@ namespace TypeChecker {
 	if (!declaration) {
 		// TODO: clean up how we build error reports
 		return {
-		    "ERROR @ line " + std::to_string(ast->m_token->m_line0 + 1) +
+		    "at line " + std::to_string(ast->m_token->m_line0 + 1) +
 		    " : accessed undeclared identifier '" + ast->text().str() + "'"};
```

Comment:
Perhaps allowing `sstream` in our error creation will be beneficial for our next iterations.

Reply:
I would like to integrate the `Log` stuff with `ErrorReport` somehow... I dunno